### PR TITLE
EPA-42: Added workflow for releasing the library

### PR DESCRIPTION
**Pull request by francogrbac-oviva**

### CODE REVIEW

**Review by demianjaeger-oviva — APPROVED** [review]
Nice, looks good from a pipeline perspective 🎉

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          echo "version: '$VERSION'"
+
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "ARTIFACT_ID=$NAMESPACE" >> $GITHUB_ENV
+      - name: Update Version
+        run: |
+          echo "updating version to '$VERSION'"
+          mvn -B versions:set "-DnewVersion=$VERSION"
+      - name: Publish package
+        run: mvn --batch-mode deploy
```
Comment (thomasrichner-oviva) [inline .github/workflows/release.yml:33]:
```suggestion
        run: mvn --batch-mode verify deploy
```
Or does deploy already verify/test?

**Review by thomasrichner-oviva — APPROVED** [review]
Very noice :D

File: .github/workflows/release.yml (hunk 1 above)
Comment (francogrbac-oviva) [reply .github/workflows/release.yml:33]: deploy already does verify

**Pull request merged**